Modules/posixmodule.c

```diff
@@ -2542,7 +2542,7 @@ os_lstat_impl(PyObject *module, path_t *path, int dir_fd)
 os.access -> bool
 
     path: path_t
-        Path to be tested; can be string or bytes
```

Comment:
Could you add a `.` at the end of this sentence?  :)

`Path to be tested; can be string or bytes.`

Reply:
Sure! :)